src/main/java/org/apache/felix/scr/impl/ScrShellCommand.java

```diff
@@ -63,12 +63,12 @@ public String getShortDescription()
     public void execute(String commandLine, PrintStream out, PrintStream err)
     {
         // Parse the commandLine to get the OBR command.
-        String[] st = commandLine.split("//s");
+        String[] st = commandLine.split("\\s+");
```

Comment:
I see a regexp error but it seems a different regexp?

    java.util.regex.PatternSyntaxException: Illegal repetition range near index 15
    [Server:AIO] (\S\S{0,}|.{1,0})(\s+|$)
    [Server:AIO]                ^

   [Server:AIO]    at java.util.regex.Pattern.compile(Pattern.java:1028) [rt.jar:1.8.0_172]
   [Server:AIO]    at org.apache.karaf.shell.support.table.Col.wrap(Col.java:178)
   [Server:AIO]    at org.apache.karaf.shell.support.table.Col.getContent(Col.java:147)